performance-tests/bench/legacy_plotting/README

```diff
@@ -1,9 +1,39 @@
 To generate new data and image files based on a scoreboard test run directory:
```

Comment:
I'd especially appreciate feedback on the README, since someday someone else may need to know how to do this.

Reply:
Is this README intended for Centipede, local user runs, or both?

Reply:
We don't really have a good process for this. Currently, the test runs are on centipede but I scp the test run directory to my local machine before running these scripts so I can update a local checkout of the gh-pages branch.